various-db-engines-it/pom.xml

```diff
@@ -36,12 +36,12 @@
     <dependency>
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
-      <version>42.1.4</version>
+      <version>42.2.2.jre7</version>
```

Comment:
Any reason to use `42.2.2.jre7` and not just `42.2.2`?

Reply:
Because I used $ mvn versions:use-latest-versions, and I didn't check that. Thanks! I will fix that.